src/Servers/Connections.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -19,5 +19,5 @@ Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionBuilder.Use(S
 Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext
 Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext.MultiplexedConnectionContext() -> void
 Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionDelegate
-abstract Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext.AcceptAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.ConnectionContext!>
+abstract Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext.AcceptAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.ConnectionContext?>
```

Comment:
Nullable "breaking change" (although the package is experimental) @pranavkm

Reply:
This [wasn't public in .NET 5](https://github.com/dotnet/aspnetcore/pull/27506), so we should be completely fine here.